utils/warnings.ml

```diff
@@ -531,8 +547,8 @@ let letter_alert tokens =
       in
       let message =
         Format.asprintf
-          "@[<v>@[Setting a warning with single lowercase \
-           or uppercase letters, like '%c' or '%c',@ is deprecated.@]@ \
+          "@[<v>@[Setting a warning with a sequence of lowercase \
+           or uppercase letters, like '%c%c',@ is deprecated.@]@ \
```

Comment:
This doesn't look right. If I understand correctly, passing the specifier `foo` would previously result in `like 'f' or 'F'`, which made sense (iirc. we didn't know whether the user used the uppercase or lowercase form?), and now it says `like fF`, when it should be something like `fo` or `foo`.

Reply:
Using the first chunk of consecutive unsigned letters seemed simpler and clearer indeed.